src/rebuild/srv.c

```diff
@@ -674,14 +668,17 @@ rebuild_leader_status_check(struct ds_pool *pool, uint32_t map_ver,
 			break;
 		}
 
+		now = ABT_get_wtime();
 		/* print something at least for each 10 secons */
 		if (now - last_print > 10) {
 			last_print = now;
 			D_PRINT("%s", sbuf);
 		}
+		dss_ult_sleep(rgt->rgt_ult, RBLD_BCAST_INTV);
+ 	}
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs